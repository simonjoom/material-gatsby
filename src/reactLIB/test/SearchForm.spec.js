import React from 'react';
import { shallow } from 'enzyme';
import SearchForm from '../src/SearchForm';

let wrapper = shallow(<SearchForm />);

describe('<SearchForm />', () => {
  test('should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
