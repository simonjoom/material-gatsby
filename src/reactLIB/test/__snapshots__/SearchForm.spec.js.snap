// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> should render 1`] = `
<form>
  <div
    className="input-field"
  >
    <input
      id="search"
      required={true}
      type="search"
    />
    <label
      htmlFor="search"
    >
      <Icon>
        search
      </Icon>
    </label>
    <Icon>
      close
    </Icon>
  </div>
</form>
`;
